README.md

```diff
@@ -60,7 +60,6 @@ func testFIFOQueueOrdering() async {
             }
         }
 
-        nonisolated
```

Comment:
@dfed should we update this too?

https://github.com/dfed/swift-async-queue/blob/6837ecbeb394eb1fc7ec6b6a858636074309abd9/Sources/AsyncQueue/ActorQueue.swift#L43-L46

Reply:
Yes we should! Good catch. I'll push this change up directly.